lua/aerial/backends/treesitter/extensions.lua

```diff
@@ -139,7 +139,7 @@ M.ruby = {
     local method = (utils.get_at_path(match, "method") or {}).node
     if method then
       local fn = get_node_text(method, bufnr) or "<parse error>"
-      if fn ~= "before" and fn ~= "after" then
+      if fn ~= item.name then
```

Comment:
When there is no argument, `@method` and `@name` are the same. This implementation is both simpler than listing every method, and fixes optional arguments in `before` and `after`.